ddtrace/contrib/asyncio/helpers.py

```diff
@@ -72,3 +73,54 @@ def _wrap_executor(fn, args, tracer, ctx):
     # fn() will be executed outside the asyncio loop as a synchronous code
     tracer._context_provider._local.set(ctx)
     return fn(*args)
+
+
+_orig_create_task = None
+
+
+def enable_task_linking():
```

Comment:
k will do.  I think the most useful scenario is going to be the case where it propagates so would you agree we should still provide an enable_task_linking method?  Because if not 
I guarantee there's going to be 1000 versions of this function in client code.